lib/rubygems/specification.rb

```diff
@@ -112,6 +112,31 @@ class Gem::Specification < Gem::BasicSpecification
 
   VALID_NAME_PATTERN = /\A[a-zA-Z0-9\.\-\_]+\z/ # :nodoc:
 
+  DEFAULT_GEMS_LIST = %w[
+    bigdecimal
+    cmath
+    csv
+    date
+    dbm
+    etc
+    fcntl
+    fiddle
+    fileutils
+    gdbm
+    io-console
+    ipaddr
+    json
+    openssl
+    psych
+    rdoc
+    scanf
+    sdbm
+    stringio
+    strscan
+    webrick
+    zlib
+  ] #:nodoc:
```

Comment:
do we want to freeze this list, or otherwise append to it when we see more gems in the default directory?